src/sass/README.md

```diff
@@ -0,0 +1,106 @@
+Sass Guidelines
+===============
+
+### General Guidelines
+In order to improve the clarity, quality and development time it is worth considering the following principles whenever possible:
+- [Keep Sass Simple](https://www.sitepoint.com/keep-sass-simple/), which means [KISS (Keep It Simple, Stupid)](https://en.wikipedia.org/wiki/KISS_principle) may override [DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don't_repeat_yourself) in some cases
+- [Single responsibility selectors](https://en.bem.info/methodology/css/#single-responsibility-principle)
+
+---
+
+### Style Guide
+
+- [Airbnb CSS / Sass Styleguide](https://github.com/airbnb/css/blob/master/README.md) is partially being followed in our code base.
+
+- [CSS with BEM](https://en.bem.info/methodology/css/) is partially being followed in our code base.
+
+- Most styling issues will be caught by [stylelint](https://github.com/stylelint/stylelint/blob/master/README.md), so before pushing your changes remember to run `grunt stylelint` to catch and fix any issues that it finds.
+
+- Check below for the rules that are not caught by styling but should be followed.
+
+### Naming Conventions
+
+<a id="naming-conventions-selectors"></a>
+**[Selectors:](#naming-conventions-selectors)** Selectors should follow the [BEM Two Dashes style](https://en.bem.info/methodology/naming-convention/#two-dashes-style): `block-name__elem-name--mod-name--mod-val`.
+
+```scss
+.button {}
+.button--disabled {}
+```
+
+Remember to follow the [Single responsibility principle](https://en.bem.info/methodology/css/#single-responsibility-principle).
+
+<a id="naming-conventions-variables"></a>
+**[Variables:](#naming-conventions-variables)** Sass variables should be in uppercase and have a meaningful prefix.
+
+```scss
+$COLOR_RED: #e31c4b;
+
+// Light theme
+$COLOR_LIGHT_BLACK_1: rgba(0, 0, 0, 0.8);
+
+// Dark theme
+$COLOR_DARK_BLUE_1: #0b0e18;
+```
+
+Keep all common variables in the [constants.scss](https://github.com/binary-com/binary-static/blob/master/src/sass/app_2/_common/base/constants.scss) file.
+
+---
+
+### Units
+
+<a id="units-flexibility"></a>
+**[Flexibility:](#units-flexibility)** If flexibility is needed, for example for font-size, use units such as `rem`, `vh`, `vw`, `fr`, and only use `px` if it's supposed to be a fixed value.
```

Comment:
I can live without it if no one agrees but I find using scalable rems as a base unit a convenient way of making a responsive layout 😄
```
html {
    font-size: 62.5%; /* 1rem = 10px */
    box-sizing: border-box;
    @include respond(big-desktop) {
        font-size: 75.5%; /* 1rem = 12px */
    }
    @include respond(tab-land) {
        font-size: 50%; /* 1rem = 8px */
    }
    @include respond(tab-port) {
        font-size: 47%; /* 1rem = 7.5px */
    }
    @include respond(phone-xl) {
        font-size: 38%; /* 1rem = 6px */
    }
    @include respond(phone-lg) {
        font-size: 35%; /* 1rem = 5.5px */
    }
}
```
(but can still use the other units for exceptions)

Reply:
@OskarAhl Actually this is what I discussed with Negar and Kelly. my suggestion is we use `rem` and components will be relative to the root. then inside the container, we use `em` which will be relative to its component.

Reply:
thanks @OskarAhl, will add this to my PR :)